File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/ProjectInformations/Dto/ProjectInformationDto.cs
using doan.ProjectManagement.Enum;
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.ProjectInformations.Dto
{
    public class ProjectInformationDto : EntityDto<Guid>
    {
        public Guid? ProjectId { get; set; }
        public string ProjectName { get; set; }
        public Guid? StudentGroupId { get; set; }
        public string StudentGroupName { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? ExpiredDate { get; set; }
        public string EmailContact { get; set; }
        public double? Mark { get; set; }
        public string Source { get; set; }
        public Status Status { get; set; }
    }
}

File: applications/teacher-web/src/app/shared/services/project/project-information/models/project-information-dto.ts
import { EntityDto } from '@abp/ng.core';
import { Status } from './status';

export class ProjectInformationDto extends EntityDto<string>{

  projectId: string;
  projectName: string;
  studentGroupId: string;
  studentGroupName: string;
  startDate: Date;
  expiredDate: Date;
  emailContact: string;
  mark: number;
  source: string;
  status: Status;

  constructor(initial: Partial<ProjectInformationDto> = {}) {
    super(initial);
  }
}
